src/utils/index.ts

```diff
@@ -16,3 +20,39 @@ export function bleCharacteristicToIconName(char: string): string {
       return 'ab-testing';
   }
 }
+
+export async function getOrCreateRealtimeRecord(): Promise<RealtimeDataModel> {
+  const collection = dataBase.get<RealtimeDataModel>('realtime_data');
+
+  try {
+    const record = await collection.find(REALTIME_DATA_ID);
+    return record;
+  } catch (err) {
+    // not found
+    // create record on first boot.
+    return dataBase.write(async () => {
+      return collection.create(r => {
```

Comment:
Does create populate the record with `null`s or `0`s? I guess it depends on the data type, hopefully for numbers its `0`s rather than `null`

Reply:
Yep it create's it as `0`.